specification/cdn/resource-manager/Microsoft.Cdn/stable/2017-10-12/examples/Policy_Create.json

```diff
@@ -0,0 +1,231 @@
+                    "matchValue":[
+                        "randomString"]
+                }
+            },
+            {
+                "name": "RequestHeader",
+                "parameters":{
+                    "@odata.type":"#Microsoft.Azure.Cdn.Models.DeliveryRuleUrlFileExtensionConditionParameters",
+                    "operator":"Contains",
+                    "selector":"Accept-Encoding",
+                    "negateCondition": false,
+                    "matchValue":[
+                        "gzip"]
+                }
+            },
+            {
+                "name": "RequestBody",
+                "parameters":{
+                    "@odata.type":"#Microsoft.Azure.Cdn.Models.DeliveryRuleUrlFileExtensionConditionParameters",
+                    "operator":"Equal",
+                    "matchValue":[
+                        "test"
+                    ]
+                }
+            },
+            {
+                "name": "UrlPath",
+                "parameters":{
+                    "@odata.type":"#Microsoft.Azure.Cdn.Models.DeliveryRuleUrlFileExtensionConditionParameters",
+                    "operator":"BeginsWith",
+                    "matchValue":[
+                        "abc"
+                    ]
+                }
+            },
+            {
+                "name": "UrlFileExtension",
+                "parameters":{
+                    "@odata.type":"#Microsoft.Azure.Cdn.Models.DeliveryRuleUrlFileExtensionConditionParameters",
+                    "operator":"EndsWith",
+                    "matchValue":[
+                        ".txt"
+                    ]
+                }
+            },
+            {
+                "name": "UrlFileName",
+                "parameters":{
+                    "@odata.type":"#Microsoft.Azure.Cdn.Models.DeliveryRuleUrlFileExtensionConditionParameters",
+                    "operator":"BeginsWith",
+                    "matchValue":[
+                        "abc"
+                    ]
+                }
+            },
+            {
+                "name": "IsMobile",
+                "parameters":{
+                    "@odata.type":"#Microsoft.Azure.Cdn.Models.DeliveryRuleUrlFileExtensionConditionParameters",
+                    "operator":"Equal",
+                    "matchValue":[
+                        "Desktop"
+                    ]
+                }
+            },
+            {
+                "name": "Cookie",
+                "parameters":{
+                    "@odata.type":"#Microsoft.Azure.Cdn.Models.DeliveryRuleUrlFileExtensionConditionParameters",
+                    "operator":"Contains",
+                    "matchValue":[
+                        "abc"
+                    ]
+                }
+            },
+            {
+                "name": "UrlScheme",
+                "parameters":{
+                    "@odata.type":"#Microsoft.Azure.Cdn.Models.DeliveryRuleUrlFileExtensionConditionParameters",
```

Comment:
>DeliveryRuleUrlFileExtensionConditionParameters [](start = 63, length = 47)

DeliveryRuleUrlSchemeConditionParameters?

Reply:
There are quite a few mis-matches. Can you double-check ?

---
In reply to: [275722479](https://github.com/hytao/azure-rest-api-specs/pull/1#discussion_r275722479) [](ancestors = 275722479)